examples/blob2d/blob_velocity.py

```diff
@@ -95,5 +94,5 @@ def blob_velocity(n,**kwargs):
         import matplotlib.pyplot as plt
         plt.plot(vx)
         plt.show()
-except:
+except ImportError:
```

Comment:
The general except is justified, as matplotlib may fail for other reasons, e.g. no suitable X11 server available.
I have changed this in some tests in the other direction, as I got errors after installing matplotlib.
Ether we want to make sure we plot - or we don't care if it fails, in which case we should ignore for whatever reason.
We might want to add an option to blob_velocity, to only plot if it is asked to plot.

Besides that, all the changes seem good, thanks for the cleanup :+1:

Reply:
That file will be getting a better cleanup soon anyway!

Adding an option to plot is something a few other examples could benefit from too.